samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties

```diff
@@ -423,6 +423,9 @@ seb_allow_spell_checking = Gaitu ortografia berrikuspena
 seb_config_upload = Safe Exam Browser konfiguratzeko fitxategia
 seb_config_upload_success = Ongi igo da konfigurazioko fitxategia
 seb_config_upload_error = Ezin izan da konfigurazioko fitxategia igo. Ziurtatu .seb luzapena duela eta ez dagoela enkriptatuta.
+seb_config_key=Allowed config key
+seb_config_key_info=The config key can only be set, after the assessment is published. You will need to download the (altered) configuration \
+again and insert the exam keys based on that file.
```

Comment:
Hi @PabloSanRoman! Could you help me with those new translations please?
The `seb_exam_keys_recommended` property also got updated, could you apply it as well?
![image](https://github.com/sakaiproject/sakai/assets/53173679/a13ab8f7-5c7d-4d0d-99fa-d825ed54ff5a)
```
seb_exam_keys_recommended=Azterketara sartzeko pasahitza jartzea gomendagarria da, Safe Exam Brower-en konfigurazioa agian ez baita automatikoki balidatzen.
```

Reply:
Hi @stetsche  , here!: 

seb_exam_keys_recommended=Pasahitz bat ezartzea gomendagarria da, konfigurazioan edo azterketetan, Safe Exam Browser tresnaren konfigurazioa ezin delako automatikoki baliozkotu.

Reply:
Thanks @PabloSanRoman! Could you also translate the new properties, please?
```
seb_config_key=Allowed config key
seb_config_key_info=The config key can only be set, after the assessment is published. You will need to download the (altered) configuration \
again and insert the exam keys based on that file.
```

Reply:
Hi @stetsche , you are welcome!: 

seb_config_key=Baimendu konfigurazioko pasahitza
seb_config_key_info=Konfigurazioko pasahitza ezarri daiteke soilik azterketa publikatu ondoren. Aldatutako konfigurazioa 
beharko duzu deskargatu berriz. Gero sartu azterketaren pasahitza, fitxategi horrekin zerikusia duena.

Reply:
Nice, thanks @PabloSanRoman.